app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CustomerItemsController < ApplicationController
+  def create
+    CustomerItem.create(customer_id: params[:customer_id], item_id: params[:item_id])
```

Comment:
Nice work creating this joins record in a separate CustomerItemsController! This is the most RESTful option :)